### PR TITLE
add qa and result

### PR DESCRIPTION
**Pull request by pohan-chi**

### CODE REVIEW

**Comment by pohanchi** [comment]
this is a template for albert qa(squad). the directory transformers/new_template     examples/qa_albert/template  the true tree structure you can refer to https://github.com/pohanchi/huggingface_albert, test squad 1.1 on albert_base and albert_xlarge is very close to the original paper

**Comment by LysandreJik** [comment]
Hi @pohanchi, great work on your ALBERT implementation! We're in the process of merging #1683 which already implements ALBERT, and would love your opinion on the implementation.

**Comment by pohanchi** [comment]
Because I don’t know how to upload the file to your team server(
https://amazonS3....), so something roughly method I use is that using
drive to store some big file (pytorch model state dict)and let people
download by themselves from drive. I just want to know that whether to do
that. By the way, I am a beginner of coderev .what is the converge mean ?
And the bigger is good or other XD.

**Comment by pohanchi** [comment]
My Repository is using lamb optimizer to train albert model. If you use lamb optimizer, make sure adam_epsilon set up to 1e-12 and weight decay set to 0.1 this will help model become stable in training

**Comment by pohanchi** [comment]
also the learning rate need to small than 5e-5 
if large, sometimes performance will drop.

**Comment by LysandreJik** [comment]
That's nice to know, thanks @pohanchi

**Comment by thomwolf** [comment]
Closing this since ALBERT is now in the library.
